modules/ROOT/pages/getting-started.adoc

```diff
@@ -213,6 +213,57 @@ CAUTION: You must stop all connector instances in a group before modifying the r
 The following commands are specific to the solo and distributed modes.
 xref:autonomous-operations.adoc[Autonomous Operations mode] has its own separate commands for managing checkpoints.
 
+[#checkpoint-command-config]
+=== Configuring the checkpoint management commands
+
+The checkpoint management commands use the same config file as the connector.
+The `--config` argument tells a checkpoint management command the filesystem path of the config file to use.
+For example:
+
+    cbes-<command> --config <path/to/connector/config.toml>
+
+If the `--config` argument is not specified, the path defaults to `config/default-connector.toml`.
+
+When running a checkpoint management command from an environment where the connector is already installed and configured, use the same config file as the connector.
+
+If you want to run a checkpoint management command in a different environment, you'll need to:
+
+1. Get the connector distribution archive and unzip it.
```

Comment:
For rendering an ordered list in Asciidoc, use a . character instead of a number.

```suggestion
. Get the connector distribution archive and unzip it.
. Edit the `config/example-connector.toml` and `config/secrets/couchbase-password.toml` files to match the settings of the connector whose checkpoints you want to manage.
```

Reply:
Nice trick! It renders the same wither way, but with just a dot you don't need to worry about updating the numbers if you add or remove an item later, so.... done!